soccer/gameplay/plays/defense/DefendClear.py

```diff
@@ -0,0 +1,35 @@
+import standard_play
```

Comment:
These plays files might have been added on accident. Is this supposed to be here?